js/components/AudioPlayer.react.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * AudioPlayer
+ */
+
+import { asyncPlayAudio, asyncPauseAudio } from '../actions/AppActions';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+class AudioPlayer extends Component {
+	render() {
+		const { dispatch, src } = this.props;
```

Comment:
Purely cosmetic question, but are you using soft tabs or real tabs here?  Just noticed the indent is different from the rest of the project, which has 2 space soft tabs.